tests/tools/importexport/migration/test_v08_to_v09.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Test export file migration from export version 0.8 to 0.9"""
+from aiida.backends.testbase import AiidaTestCase
+from aiida.tools.importexport.migration.utils import verify_metadata_version
+from aiida.tools.importexport.migration.v08_to_v09 import migrate_v8_to_v9
+
+from tests.utils.archives import get_json_files
+
+
+class TestMigrate(AiidaTestCase):
+    """Test migration of export files from export version 0.8 to 0.9"""
+
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
+        super().setUpClass(*args, **kwargs)
+
+        # Utility helpers
+        cls.external_archive = {'filepath': 'archives', 'external_module': 'aiida-export-migration-tests'}
+        cls.core_archive = {'filepath': 'export/migrate'}
+
+    def test_migrate(self):
```

Comment:
```suggestion
    def test_migrate_v8_to_v9(self):
```
Same as the comment above.